src/SwipeProfiles.js

```diff
@@ -137,64 +137,77 @@ class SwipeProfiles extends React.Component {
   }
 
   render() {
+    
+    const id = this.state.id;
     const potentialFriends = this.state.potentialFriends;
     const displayedUserId = this.state.displayedUserId;
     const error = this.state.error;
+
```

Comment:
is there any way to save this somewhere and reuse it since it's on almost every page?

Reply:
It's saved in the cookies. When it sets the state initially is calls the getCookies method to retrieve the value. So it's saved in the cookies so we can use it on every page

Reply:
Oh I thought you meant the id value, okay I can create a method that does that